lib/ueberauth.ex

```diff
@@ -194,7 +194,7 @@ defmodule Ueberauth do
   @doc false
   def init(opts \\ []) do
     {provider_list, opts} = Keyword.pop(opts, :providers, :all)
-    opts = Keyword.merge(Application.get_env(:ueberauth, Ueberauth), opts)
+    opts = Keyword.merge(Application.fetch_env!(:ueberauth, Ueberauth), opts)
```

Comment:
```suggestion
    opts = 
      :ueberauth
      |> Application.fetch_env!(Ueberauth)
      |> Keyword.merge(opts)
```

Reply:
`|>` pipe operator does not mean transformation in any possible way, it is weird that you correlate the pipe operator with transformation, to be honest. 

Also this deicision is based on the popular for linting `credo`, right now it is disabled, so this is not like my personal opinion only.

Also,

`fetch_env/2` transform two inputs into a keyword list of configuration.
`merge/2` transform two inputs into a single input by merging them together.

It is a matter of wording 😄 

So what is your definition of transformation?

> BUT, if you do want this style I'm willing to make the commit.

Yes, please I would prefer if you change it since we will be doing this in the future anyway.

> It is your library after all. :)

I would say more our library. Yes, I take the responsibility to maintain it in some cases but do not feel we are authoritarian and everything have to be the way `I` want it, we are a team.

Reply:
Also, this decision is based on the popular for linting `credo`, right now it is disabled, so this is not like my personal opinion only.

Also,

`fetch_env/2` transform two inputs into a keyword list of configuration.
`merge/2` transform two inputs into a single input by merging them together.

It is a matter of wording 😄 

So what is your definition of transformation?

> BUT, if you do want this style I'm willing to make the commit.

Yes, please I would prefer if you change it since we will be doing this in the future anyway.

> It is your library after all. :)

I would say more our library. Yes, I take the responsibility to maintain it in some cases but do not feel we are authoritarian and everything have to be the way `I` want it, we are a team.

Reply:
Here the basic idea I have (if you're still interested in it). I reserve `|>` for when there is an obvious main argument being transformed... for instance:

```elixir
conn |> assign(:hello, "world")
```
I see `conn` is the thing being transformed, as we are adding an assignment to the `conn` struct. You COULD describe this as a transformation of the atom `:hello` or the string "world", but that feels like an odd way of phrasing it to me.

But I'm not going to try to impose these conventions on you (or anyone else). The aren't hills I would want to die on. Haha.